sys/vm/pmap.h

```diff
@@ -170,6 +169,9 @@ void		 pmap_remove_all(vm_page_t m);
 void		 pmap_remove_pages(pmap_t);
 void		 pmap_remove_write(vm_page_t m);
 void		 pmap_sync_icache(pmap_t, vm_offset_t, vm_size_t);
+#ifdef CPU_CHERI
```

Comment:
MI bits should be using `__has_feature(capabilities)` (like in the previous hunk's context).